### ISSUE TITLE
The sample does not work

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5-build-decoupled-document#8]
The code of the sample:

```js
DecoupledEditor.create( editorData )
```

But in `ckeditor.js` it is `DecoupledDocumentEditor`

**Comment by jodator** [comment]
ps.: I've run the `npm run build` script and it does fix something:

![selection_013](https://user-images.githubusercontent.com/247363/38420509-3b8b9dec-39a4-11e8-9c30-5ce0004ee517.png)

But the translations got messed up:
![selection_014](https://user-images.githubusercontent.com/247363/38420557-6935932e-39a4-11e8-9ef9-08625fda839e.png)


Error in the console:
```
ckeditor.js:7 Uncaught TypeError: CKEDITOR_TRANSLATIONS.add is not a function
    at ckeditor.js:7
(anonymous) @ ckeditor.js:7
```

**Comment by ma2ciek** [comment]
Looks like it uses old translation service. I'll check it.

**Comment by Reinmar** [comment]
Fixed with fd0f69ad.
